db/db_block_cache_test.cc

```diff
@@ -1714,6 +1714,13 @@ TEST_P(DBBlockCacheKeyTest, StableCacheKeys) {
 #endif  // !ROCKSDB_LITE
 
   Close();
+#ifndef ROCKSDB_LITE
+  // Delete the external files
+  for (const auto& e : external) {
+    Status s = options.env->DeleteFile(e);
+    ASSERT_TRUE(s.IsNotFound() || s.ok());
```

Comment:
When will `s` be non-ok?

Reply:
If the file does not exist, the I presume s.IsNotFound would be returned.  I do not know if that would ever happen or not.